packages/notte-agent/src/notte_agent/common/config.py

```diff
@@ -38,6 +38,14 @@ class AgentConfig(FrozenConfig, ABC):
     reasoning_model: str = Field(
         default=LlmModel.default(), description="The model to use for reasoning (i.e taking actions)."
     )
+    @validator("reasoning_model", pre=True)
+    def convert_model(cls, value):
```

Comment:
can you also add the same check in `AgentRunRequest` that could also be beneficial there